docker/app-base/entrypoint.sh

```diff
@@ -4,7 +4,10 @@
 /etc/init.d/rsyslog start
 
 # run lfmergeqm on startup to clear out any failed send/receive sessions from previous container
-which lfmergeqm && su www-data -s /bin/bash -c lfmergeqm
+if [ which /lfmergeqm-background.sh ]; then
```

Comment:
`which` is not present in this image

Reply:
@rmunn sounds like the lfmergeqm isn't working right off the bat.  @longrunningprocess I have seen the errors in the log you mention and thought "we should fix those".  I have created an issue to track the errors in the log #1055

Reply:
*Boggle*. What kind of Docker image doesn't have `which` in it? I understand not having `ping`, but Busybox implements `which`. I just assumed it would be there, as a tool I could use in my scripts to verify that a *different* command exists before I run it.

I'll see what I can do about tweaking that script to run in the absence of a `/bin/which` command.

Reply:
The Docker image *does* have `which`, I just messed up the syntax. Instead of `if [ which some-command ]; then`, I needed to omit the `[ ]` and write `if which some-command; then`.

Reply:
https://github.com/sillsdev/web-languageforge/pull/1059 fixes the `[: which: unexpected operator` issue. I'll make a separate PR for the `imklog` issue.

Reply:
Changed my mind and put both fixes into #1059 since they're both small and easy to review. So that one PR will fix both of these issues.

Reply:
My important point was simply to remove the conditional altogether, you're installing `lfmergeqm-background.sh` in the Dockerfile already, seems silly to me to check to make sure it's there.